tox.ini

```diff
@@ -25,15 +26,20 @@ commands =
     python setup.py check -m -r -s
 
 [testenv:pep8]
-basepython = python2.7
-deps = flake8==2.3.0
+basepython = python2
+deps = flake8==3.3.0
 commands = flake8 .
 
 [testenv:py3pep8]
-basepython = python3.3
-deps = flake8==2.3.0
+basepython = python3
+deps = flake8==3.3.0
 commands = flake8 .
 
+[testenv:isort]
```

Comment:
It's a tox command because this is a check equivalent to pep8 - a linting check/test.